lib/internal/structured_clone.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const {
+  codes: {
+    ERR_INVALID_ARG_TYPE,
+    ERR_MISSING_ARGS,
+  },
+} = require('internal/errors');
+const webidl = require('internal/webidl');
+
+const {
+  structuredClone: nativeStructuredClone,
+} = internalBinding('messaging');
+
+webidl.converters['sequence<object>'] = webidl.createSequenceConverter(webidl.converters.any);
```

Comment:
I think this should be in `lib/internal/webidl` instead to avoid surprises if other files try to reuse it in the future.